pkg/config/config.go

```diff
@@ -66,7 +66,7 @@ type StaticConfig struct {
 func Default() *StaticConfig {
 	return &StaticConfig{
 		ListOutput: "table",
-		Toolsets:   []string{"core", "config", "helm"},
+		Toolsets:   []string{"core", "config", "helm", "openshift-core"},
```

Comment:
I'm not sure we want to add this to the default config here

On the one hand, it makes sense to have the openshift tools enabled by default in the openshift fork

However, this introduces lots of changes to the other config/toolset tests as now they all expect the `openshift-core` toolset to be there

I'm worried this will lead to a load of conflicts whenever we make changes to those upstream...

Reply:
The second commit in #396 should take care of the failing tests.